docs/concepts/admission-controller.md

```diff
@@ -12,4 +12,42 @@ This handler checks if the incoming request contains a Kubernetes secret with a
 
 Namespaces are the backing entities of Gardener projects in which shoot clusters objects reside.
 This validation handler protects active namespaces against premature deletion requests.
-Therefore, it denies deletion requests if a namespace still contains shoot clusters or if it belongs to a non-deleting Gardener project (w/o `.metadata.deletionTimestamp`).
\ No newline at end of file
+Therefore, it denies deletion requests if a namespace still contains shoot clusters or if it belongs to a non-deleting Gardener project (w/o `.metadata.deletionTimestamp`).
+
+### Resource Size Validator
+
+Since users directly apply Kubernetes native objects to the Garden cluster, it also involves the risk of being vulnerable to DoS attacks because these resources are read by controllers in the Gardener landscape.
```

Comment:
```suggestion
Since users directly apply Kubernetes native objects to the Garden cluster, it also involves the risk of being vulnerable to DoS attacks because these resources are continuously watched and read by controllers.
```